index.html

```diff
@@ -21,53 +21,64 @@
         <link rel="preconnect" href="https://fonts.googleapis.com">
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <link href="https://fonts.googleapis.com/css2?family=Inter:wght@500&display=swap" rel="stylesheet">
+    <!-- Font awesome - Icons -->
+    <script src="https://kit.fontawesome.com/841b60bdd5.js" crossorigin="anonymous"></script>
 
     <title>Daniel`s Portafolio</title>
 
 </head>
 <body>
-    
+    <main>
     <header>
-        <a href="index.html" id="logo">Jhon Doe</a>
         <nav>
-            <img id="menu" src="bars-solid.svg" alt="menu icon">
+            <a href="index.html" id="logonav">Jhon Doe</a>
```

Comment:
Great project @dgonzalesi 🤝

- [ ] [Optional] You may please make this logo your name since it is your portfolio. There is no need of using John Doe.